src/main/java/de/dentrassi/rpm/builder/YumMojo.java

```diff
@@ -150,11 +156,15 @@ public void execute () throws MojoExecutionException, MojoFailureException
 
             int count = 0;
 
-            final Set<Artifact> deps = this.project.getArtifacts ();
-            if ( deps != null )
+            if ( !this.skipDependencies )
             {
-                deps.forEach ( art -> addPackage ( creator, art.getFile () ) );
-                count++;
+                this.logger.debug ( "Skipping rpm artifacts from maven dependencies" );
```

Comment:
I think the message is wrong here. It should be moved into the "else" branch. Also could you uppercase "RPM".